### DISCUSSION TITLE
Questions for Wannierberri using FPLO hamdata calculating AHC

### DISCUSSION THREAD

**Discussion by wyb1999** [wannier-berri/wannier-berri#210] [Q&A]
For FPLO calculating AHC, I have tried to use hamdata calculated from autoprojected wannierfunction. However, I couldn't get similar data as VASP. It's very far from the result from VASP. Are there still any error in FPLO calculation? What's more, what's the unit of AHC calculating from FPLO? Looking forward to your response!

**Comment by stepan-tsirkin** [comment]
@wyb1999 , sorry for a late reply, unfortunately I did not notice your question earlier. 

The units of AHC in WannierBerri is always S/m (from version 13.0 or so) , no matter for which system it is calculated.  Note, It is different from postw90.x, which uses S/cm.

The results may be somewhat different due to different bandstructures obtained, but the physics should be the same.  For which system are you calculating? Did you make sure that results are converged, and proper Fermi levels are set. Could you show some figures of what you get with different codes?
